tests/iter/titerable.nim

```diff
@@ -6,7 +6,7 @@ discard """
 8
 12
 '''
-  disabled: "true"
+    disabled: "true"
```

Comment:
Created in 2014-03 in a commit that references #966
Disabled around 2015-11 citing 'lambda lifting support for iterToProc plugin'

Reply:
Known issue

Reply:
Do we have a issue number?

Reply:
nope, I'd go with textual description, lambda lifting and iterators are likley to go through a bit spec definition and this will either get folded in and it's a good signal to know whether we fixed something by happenstance or not.